packages/default-plugins/src/ed25519/keypair.ts

```diff
@@ -45,17 +46,27 @@ export class EdKeypair implements DidableKey, ExportableKey {
     return ed25519.sign(this.secretKey, msg)
   }
 
-  async export(format: Encodings = "base64pad"): Promise<string> {
+  async export(): Promise<PrivateKeyJwk> {
     if (!this.exportable) {
       throw new Error("Key is not exportable")
     }
 
-    return uint8arrays.toString(this.secretKey, format)
+    const jwk: PrivateKeyJwk = {
+      kty: "EC",
+      crv: "Ed25519",
+      d: uint8arrays.toString(this.secretKey, "base64pad"),
+    }
+    return jwk
   }
```

Comment:
Thanks for also aligning the `EdKeypair` to export as `PrivateKeyJwk`.
Do you mind adding a comment with a reference/link to something that specifies/explains why the format should be `kty`/`crv`/`d`? :pray: